Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ubuntu
+FROM python:3.9.0-buster
 
-COPY requirements.txt requirements.txt
+RUN apt-get update
 
-RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
-RUN pip3 install -r requirements.txt
+RUN pip3 install Pillow
 RUN git clone https://github.com/tensorflow/tflite-micro.git
 
 WORKDIR tflite-micro
 RUN make -f tensorflow/lite/micro/tools/make/Makefile hello_world_bin
-RUN make -f tensorflow/lite/micro/tools/make/Makefile test_hello_world_test
```

Comment:
If this is conflicting to run hello_world_bin, line11 could be commented out temporarily now? This would be need for coverage testing later

Reply:
Added tests, was unsure if they were meant to be in the dockerfile